cmd/build.go

```diff
@@ -46,6 +46,16 @@ https://docs.snyk.io/products/snyk-infrastructure-as-code/custom-rules/getting-s
 		}
 		return nil
 	},
+	PostRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
what does the capital `E` stand for?

Reply:
Based on https://github.com/spf13/cobra/blob/master/user_guide.md#returning-and-handling-errors, `E` means that the command can return an error to the user. We are just printing so I changed this to `PostRun` - good question!